packages/xo-server-auth-ldap/src/test-cli.js

```diff
@@ -28,9 +30,15 @@ execPromise(async args => {
     }
   )
 
-  const plugin = createPlugin({
-    logger: console.log.bind(console),
-  })
+  configure([
+    {
+      filter: process.env.DEBUG ?? 'xo:xo-server-auth-ldap',
+      level: 'debug',
```

Comment:
You can remove this entirely, as long as the log matches the filter it will be shown :slightly_smiling_face: